### PR TITLE
optional highlight first element on autocomplete reset

### PR DESCRIPTION
**Pull request by jordanwnolan**
/cc @ciniglio @JarredLHumphrey @wija @hannahbarudin

### CODE REVIEW

File: src/autocomplete_list.coffee [hunk 1]
```diff
@@ -25,7 +25,7 @@ class Boneyard.AutocompleteList  extends Backbone.View
       @resultIsMulti = false
       @resultCollection.reset results
     )
-
```
Comment (ciniglio) [inline src/autocomplete_list.coffee]: Why coerce to boolean here?

Comment (jordanwnolan) [inline src/autocomplete_list.coffee]: oops I'm coercing in 2 spots here.  It'll work the same with out coercion, is that preferred?

Comment (ciniglio) [inline src/autocomplete_list.coffee]: Generally, I think so. @wija or @jarredlhumphrey?

Comment (JarredLHumphrey) [inline src/autocomplete_list.coffee]: Yea usually not needed unless you're using === comparison to bool values. The unless will coerce it to a boolean.

**Comment by JarredLHumphrey** [comment]
:+1:

**Comment by jordanwnolan** [comment]
Think this didn't get auto-merged

**Pull request closed without merging**